packages/upload-client/test/index.test.js

```diff
@@ -1,8 +1,162 @@
 import assert from 'assert'
-import Client from '../src/index.js'
+import * as Client from '@ucanto/client'
+import * as Server from '@ucanto/server'
+import * as CAR from '@ucanto/transport/car'
+import * as CBOR from '@ucanto/transport/cbor'
+import * as Signer from '@ucanto/principal/ed25519'
+import { add as storeAdd } from '@web3-storage/access/capabilities/store'
+import { add as uploadAdd } from '@web3-storage/access/capabilities/upload'
+import { uploadFile, uploadDirectory } from '../src/index.js'
+import { service as id } from './fixtures.js'
+import { randomBytes } from './helpers/random.js'
+import { File } from './helpers/shims.js'
+import { mockService } from './helpers/mocks.js'
 
-describe('index', function () {
-  it('should export a client object', () => {
-    assert(Client)
+describe('uploadFile', () => {
+  it('uploads a file to the service', async () => {
+    const res = {
+      status: 'upload',
+      headers: { 'x-test': 'true' },
+      url: 'http://localhost:9200',
+    }
+
+    const account = await Signer.generate()
+    const issuer = await Signer.generate()
```

Comment:
I found the test difficult to understand initially... a comment here would help
```suggestion
    const issuer = await Signer.generate() // The "user" that will ask the service to accept the upload
```

we can circle back and rename account -> space later.